tests/test_buffer.py

```diff
@@ -239,24 +239,31 @@ def check_identical(sarr, ndarr):
         ndarrT = ndarr.transpose()
 
         sarr = modmesh.SimpleArrayFloat64(array=ndarr)
-        noarr = sarr.transpose()
+        sarr2 = sarr.transpose()
         check_identical(sarr, ndarrT)
-        self.assertEqual(noarr, None)
+        check_identical(sarr2, ndarrT)
```

Comment:
Since now Python `transpose()` returns self, `check_identical` inner function should be renamed to `check_equal`.